addons/website_hr_recruitment/models/hr_department.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class Department(models.Model):
+    _inherit = 'hr.department'
+
+    def name_get(self):
+        # Get department name using superuser, because model is not accessible for portal users
```

Comment:
Try to line return your comment / docstrings at max 80 char (https://github.com/odoo/odoo/pull/83508#discussion_r796442104)
```suggestion
        # Get department name using superuser, because model is not accessible
        # for portal users
```